blueprints/ember-fullcalendar/index.js

```diff
@@ -2,10 +2,8 @@ module.exports = {
   normalizeEntityName: function() {},
 
   afterInstall: function() {
-    return this.addBowerPackageToProject('fullcalendar').then(() => {
-      return this.addBowerPackageToProject('fullcalendar-scheduler').then(() => {
-        return this.addAddonToProject('ember-cli-moment-shim', '0.6.2');
-      });
+    return this.addBowerPackageToProject('fullcalendar', '^2.7.3').then(() => {
```

Comment:
Re bower. I did some research/asking yesterday. With bower dependencies, an addon's bower dependencies don't get installed into the `/bower_components` directory unless you explicitly do it with the blueprint. Then to actually include the code in `vendor.js` you do `app.import()`. The dependencies listed in `bower.json` are for your dummy app, not for users consuming your addon.